src/frontend/AppContainerWrapper.tsx

```diff
@@ -96,9 +93,9 @@ const AppContainerWrapper = () => {
   const navRef = React.useRef<NavigationContainerComponent>();
   const [inviteModalEnabled, setInviteModalEnabled] = React.useState(true);
   const [queuedInvite, setQueuedInvite] = React.useState<string | null>(null);
-  const [{ experiments }] = React.useContext(SettingsContext);
   const [hidePracticeBar, setHidePracticeBar] = React.useState(true);
   const [hidePracticeMode, setHidePracticeMode] = React.useState(false);
+  const { onboarding } = devExperiments;
```

Comment:
nit: i _think_ since this is a stable constant value, you can define this outside the React component. Can't fully remember, but extracting it out here within the component may have rendering implications, which can be avoided since it's a stable constant

Reply:
Right, i think ive just been stuck in the react world for so long, i forget how normal js works!